dev/com.ibm.ws.app.manager.springboot/resources/com/ibm/ws/app/manager/springboot/internal/resources/Messages.nlsprops

```diff
@@ -58,3 +58,7 @@ error.no.manifest.found.useraction=Configure a valid Spring Boot application, wh
 error.no.spring.class.found=CWWKC0257E: The Spring Boot application META-INF/MANIFEST.MF file must specify a Start-Class header.
 error.no.spring.class.found.explanation=Spring Boot applications must contain a main class specified by the Start-Class manifest header.
 error.no.spring.class.found.useraction=Configure a valid Spring Boot application, which contains a Start-Class manifest header, with the server.
+
+error.missing.ssl=CWWKC0258E: Could not run the application because it requires an SSL feature to be configured in the server.xml.
```

Comment:
This line is missing its subject. We also try to use the present tense as much as possible. Maybe something like, "The program cannot run the application because the application requires an SSL feature to be configured in the server.xml file."

Reply:
We have two other similar messages:
`error.need.springboot.version.20=CWWKC0253E: Could not run the application because the wrong version of the Spring Boot feature is configured in server.xml. The application requires the feature springBoot-2.0 to be configured.`

`error.missing.servlet=CWWKC0254E: Could not run the application because it requires a Servlet feature to be configured in the server.xml.`

I think we should instead use something like this for all three messages:
`The application failed to start because ...`